modules/serverless-kn-cli-plugins.adoc

```diff
@@ -0,0 +1,11 @@
+// Module is included in the following assemblies:
+//
+// serverless/installing-knative-client.adoc
+
+[id="serverless-kn-plugins_{context}"]
+= `kn` plugins
+
+The Red Hat distribution of the `kn` command contains extensions of core `kn`, called plugins. Their main use is extending `kn` to use additional commands provided on {ServerlessProductName}, for example, the Kafka stack API. Main `kn` functionality and `kn` plugins are used in the same way.
+
+Currently, Red Hat provides the `kn kafka` and the `kn func` plugins. Fow now, both plugins are Technology Previews.
```

Comment:
Remove `kn func` - it's not tech preview, just dev preview atm, so it shouldn't be included anywhere in our main docs yet.